content/2017-munich/sponsor.md

```diff
@@ -141,27 +227,27 @@ attendee tickets, please let us know in advance.
 Prometheus is a leading open-source monitoring system and time series database
 which is used by companies of all sizes for their mission-critical monitoring.
 
-We expect PromCon to attract a crowd of experienced and influential
+PromCon attracts a crowd of experienced and influential
 infrastructure engineers, both via in-person attendance and through video
-recordings that will be made available to the public after the conference.
+recordings that are made available to the public after the conference.
 **Featuring your name and brand in front of this audience will drive the
 adoption of your products and services in the infrastructure world, as well as
 present you with potential hiring opportunities!** And of course, if your
 organisation is using Prometheus already, you will benefit directly from helping
 its community to grow and thrive.
 
-To give you an idea of the overall buzz in the Prometheus ecosystem, in January
-2016, we counted:
+To give you an idea of the overall buzz in the Prometheus ecosystem, in March
+2017, we counted:
 
-* ~4800 total GitHub stars across our repositories
-* 200+ contributors
-* 2300+ pull requests (60+ open)
-* 1100+ issues (300+ open)
-* 150+ people in our IRC channel (#prometheus on FreeNode)
-* 250+ people on the mailing list who have created 300+ threads
-* 20+ Prometheus-related talks and workshops
-* 100+ articles and blog posts
-* 50+ 3rd party integrations
+* 11,000+ total GitHub stars across our repositories
+* 550+ contributors
+* 3900+ pull requests (120+ open)
+* 2500+ issues (600+ open)
+* 400+ people in our IRC channel (#prometheus on FreeNode)
+* 700+ people on the mailing list who have created 1200+ threads
+* ???+ Prometheus-related talks and workshops
```

Comment:
@brian-brazil I updated all the other stats from various sources, but I wasn't sure how we arrived at this one last time. Can you produce a new number for this (was it you?) or otherwise we can just omit it.